en/backup-to-s3.md

```diff
@@ -0,0 +1,287 @@
+
+* `.spec.metadata.namespace`: the namespace where the `Backup` CR is located.
+* `.spec.from.host`: the address of the TiDB cluster to be backed up.
+* `.spec.from.port`: the port of the TiDB cluster to be backed up.
+* `.spec.from.user`: the accessing user of the TiDB cluster to be backed up.
+* `.spec.from.tidbSecretName`: the secret of the credential needed by the TiDB cluster to be backed up.
+* `.spec.storageClassName`: the persistent volume (PV) type specified for the backup operation. If this item is not specified, the value of the `default-backup-storage-class-name` parameter (`standard` by default, specified when TiDB Operator is started) is used by default.
+* `.spec.storageSize`: the PV size specified for the backup operation. This value must be greater than the size of the TiDB cluster to be backed up.
+
+More S3-compatible `provider`s are described as follows:
+
+* `alibaba`: Alibaba Cloud Object Storage System (OSS) formerly Aliyun
+* `digitalocean`: Digital Ocean Spaces
+* `dreamhost`: Dreamhost DreamObjects
+* `ibmcos`: IBM COS S3
+* `minio`: Minio Object Storage
+* `netease`: Netease Object Storage (NOS)
+* `wasabi`: Wasabi Object Storage
+* `other`: Any other S3 compatible provider
+
+## Scheduled full backup to S3-compatible storage
+
+You can set a backup policy to perform scheduled backups of the TiDB cluster, and set a backup retention policy to avoid excessive backup items. A scheduled full backup is described by a custom `BackupSchedule` CR object. A full backup is triggered at each backup time point. Its underlying implementation is the ad-hoc full backup.
+
+### Prerequisites for scheduled backup
+
+The prerequisites for the scheduled backup is the same as the [prerequisites for ad-hoc backup](#prerequisites-for-ad-hoc-backup).
+
+### Scheduled backup process
+
++ Scheduled backup to Amazon S3.
+
+    1. In the `backup-gcs.yaml` file, edit `host`, `port`, `user`, `projectId` and save your changes.
+
+        {{< copyable "" >}}
+
+        ```yaml
+        ---
+        apiVersion: pingcap.com/v1alpha1
+        kind: BackupSchedule
+        metadata:
+        name: demo1-backup-schedule-s3
+        namespace: test1
+        spec:
+        #maxBackups: 5
+        #pause: true
+        maxReservedTime: "3h"
+        schedule: "*/2 * * * *"
+        backupTemplate:
+            from:
+            host: <tidb-host-ip>
+            port: <tidb-port>
+            user: <tidb-user>
+            secretName: backup-demo1-tidb-secret
+            s3:
+            provider: aws
+            secretName: s3-secret
+            # region: us-east-1
+            # storageClass: STANDARD_IA
+            # acl: private
+            # endpoint:
+            storageClassName: local-storage
+            storageSize: 10Gi
+        ```
+
+    2. Create the `BackupSchedule` CR to enable the scheduled full backup to Amazon S3:
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        kubectl apply -f backup-schedule-s3.yaml
```

Comment:
@onlymellb Please help confirm whether the file name `backup-schedule-s3.yaml` in the command is correct, thanks!